File: hw/astudillo/U1/HW08ClassPythonChickenFarm/ChickenFarmSimulator/ChickenFarmSystem.py
from model.Egg import EggClass
from model.Poop import PoopClass

class ChickenFarmSystem:
    egg = EggClass(1)
    egg2 = EggClass(2)
    egg3 = EggClass(3)
    poop = PoopClass(1)
    numberOfTires = 5
    name = "Gerald"

    print("This a Chicken Farm Simulator")
    print("the number of tires is ", numberOfTires)
    print("the id poop is: ", poop.id)
    print("My name is ", name)
    print("the id egg is: ", egg.id)
    print("the id egg 2 is: ", egg2.id)
    print("the id egg 3 is: ", egg3.id)

File: ws/astudillo/U1/WS05ChickenFarm/ChickenFarmSimulator/src/ec/edu/espe/farmsimulator/view/ChickenFarmSystem.java
package ec.edu.espe.farmsimulator.view;

import ec.edu.espe.farmsimulator.model.Egg;
import ec.edu.espe.farmsimulator.model.Poop;
import java.util.HashSet;

/**
 *
 * @author Gerald Astudillo SoftCrafters DCCO ESPE
 */
public class ChickenFarmSystem {

    public static void main(String args[]) {

        Poop poop;
        int numberOfTires;
        Egg egg;
        Egg egg2;
        Egg egg3;
        String name;

        poop = new Poop();
        egg = new Egg();
        egg2 = new Egg();
        egg3 = new Egg();
        numberOfTires = 5;
        name = "Gerald";

        System.out.println("This a Chicken Farm Simulator");
        System.out.println("the number of Tires is " + numberOfTires);
        poop.setId(1);
        System.out.println("the number of Poops is " + poop);
        System.out.println("the number of Poops is " + poop.getId());
        System.out.println("My Name is " + name);
        egg.setId(1);
        System.out.println("Chicken egg is " + egg);
        System.out.println("Chicken egg id is " + egg.getId());
        egg2.setId(2);
        System.out.println("Chicken egg is " + egg2);
        System.out.println("Chicken egg id is " + egg2.getId());
        egg3.setId(3);
        System.out.println("Chicken egg is " + egg3);
        System.out.println("Chicken egg id is " + egg3.getId());

    }
}
